test/e2e/elasticache/replication_group/coverage_tests/rg_replication.sh

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env bash
```

Comment:
how about `test/e2e/elasticache/replication_group/update_fail.sh`?

Reply:
Only a couple of the test cases are failure/negative tests, but many are currently failing because we have some issues with adding/removing replicas. These are "update" tests as are the scaling tests, but I wanted to separate the "update" tests into separate groups of scaling vs. replication to highlight where our problems are